### PR TITLE
[GH-1007]: Updated the instruction in the setup flow for the cloud Oauth2.0

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
#### Summary
1) Updated the step to include the sharing of the app to make it public and usable to other users in the setup command for cloud Oauth2.0.

#### Ticket Link
https://github.com/mattermost/mattermost-plugin-jira/issues/1007

#### Steps To Test
1) Run the slash command /jira setup and select cloud Oauth2.0. In the step 1: Creation of the App, You can find an additional step to make the app public.

### CODE REVIEW

File: server/setup_flow.go [hunk 1]
```diff
@@ -323,7 +323,9 @@ func (p *Plugin) stepCloudOAuthConfigure() flow.Step {
 			"       {{.OAuthCompleteURL}}\n"+
 			"8. Select **Settings** in the left menu.\n"+
 			"9. Copy the **Client ID** and **Secret** and keep it handy.\n"+
-			"10. Click on the **Configure** button below, enter these details and then **Continue**.", JiraScopes)).
+			"10. By default the app will be listed as private. If you wish to make it public means other users can use it too then Select the **Distribution** in the left menu.\n"+
```
Comment (mickmister) [inline server/setup_flow.go:326]:
```suggestion
			"10. By default the app will be created as private. In order to share it with your organization, select the **Distribution** in the left menu.\n"+
```

**Review by mickmister — COMMENTED** [review]
Great work on this @Kshitij-Katiyar :+1:  Just one suggestion for wording of the setup instruction

**Review by mickmister — APPROVED** [review]

**Comment by mickmister** [comment]
@Kshitij-Katiyar Based on the instructions here https://github.com/mattermost/mattermost-plugin-jira?tab=readme-ov-file#configuration, is there anything you think can be changed there to simplify the process of setting up the plugin? I find it odd that it mentions `/jira setup` and `/jira instance install ...` when the `setup` command already handles the instance install piece

**Comment by Kshitij-Katiyar** [comment]
> @Kshitij-Katiyar Based on the instructions here https://github.com/mattermost/mattermost-plugin-jira?tab=readme-ov-file#configuration, is there anything you think can be changed there to simplify the process of setting up the plugin? I find it odd that it mentions `/jira setup` and `/jira instance install ...` when the `setup` command already handles the instance install piece

@mickmister sure, will update that and create the PR.

**Pull request merged**